src/cluster/src/check/mod.rs

```diff
@@ -379,16 +398,208 @@ impl ClusterChecker {
             Box::new(CreateServiceAccountPermission),
             Box::new(LoadBalancer),
         ];
+        self.checks.extend(checks);
+        self
+    }
 
-        // Collect results from running checks
-        let mut results = vec![];
+    /// Adds all checks required for starting a cluster on minikube.
+    ///
+    /// Note that no checks are run until [`run`] or [`run_with_progress`].
+    pub fn with_minikube_checks(mut self) -> Self {
+        let checks: Vec<Box<(dyn ClusterCheck)>> = vec![
+            Box::new(LoadableConfig),
+            Box::new(HelmVersion),
+            Box::new(SysChart),
+            Box::new(AlreadyInstalled),
+            Box::new(LoadBalancer),
+        ];
+        self.checks.extend(checks);
+        self
+    }
+
+    /// Adds all checks required for starting a local cluster.
+    ///
+    /// Note that no checks are run until [`run`] or [`run_with_progress`].
+    pub fn with_local_checks(mut self) -> Self {
+        let checks: Vec<Box<(dyn ClusterCheck)>> = vec![
```

Comment:
Seems like this should be moved to local cluster install?

Reply:
This is just a convenience method to load up local checks into the checker. It makes it nicer for the caller, e.g. the LocalClusterInstaller. Callers can then just do something like

```rust
ClusterChecker::empty()
    .with_local_checks()
    .run_and_fix(fix)
    .await
```